Bottom: 01e0c526c77aef41cdc2ba9aedfd32930c02e6ee
Top:    32bcea264d817bc347b447007bd2efffe6153f30
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-17 03:04:11 +0100

Refresh of add-support-for-depth-to-trian

---

diff --git a/src/hid/common/hidgl.h b/src/hid/common/hidgl.h
index c6f79c4..ce4724d 100644
--- a/src/hid/common/hidgl.h
+++ b/src/hid/common/hidgl.h
@@ -38,17 +38,6 @@ void hidgl_flush_triangles (triangle_buffer *buffer);
 void hidgl_ensure_triangle_space (triangle_buffer *buffer, int count);
 
 static inline void
-hidgl_add_triangle (triangle_buffer *buffer,
-                    GLfloat x1, GLfloat y1,
-                    GLfloat x2, GLfloat y2,
-                    GLfloat x3, GLfloat y3)
-{
-  hidgl_add_triangle_3D (x1, y1, global_depth,
-                         x2, y2, gloabl_depth,
-                         x3, y3, global_depth);
-}
-
-static inline void
 hidgl_add_triangle_3D (triangle_buffer *buffer,
                        GLfloat x1, GLfloat y1, GLfloat z1,
                        GLfloat x2, GLfloat y2, GLfloat z2,
@@ -66,6 +55,17 @@ hidgl_add_triangle_3D (triangle_buffer *buffer,
   buffer->triangle_count++;
 }
 
+static inline void
+hidgl_add_triangle (triangle_buffer *buffer,
+                    GLfloat x1, GLfloat y1,
+                    GLfloat x2, GLfloat y2,
+                    GLfloat x3, GLfloat y3)
+{
+  hidgl_add_triangle_3D (x1, y1, global_depth,
+                         x2, y2, gloabl_depth,
+                         x3, y3, global_depth);
+}
+
 // void draw_grid ()
 void hidgl_draw_line (int cap, double width, int x1, int y1, int x2, int y2, double scale);
 void hidgl_draw_arc (double width, int vx, int vy, int vrx, int vry, int start_angle, int delta_angle, double scale);
